owl_msgs/action/PanTiltMoveCartesian.action

```diff
@@ -0,0 +1,7 @@
+# goal definition
+int32 frame
+geometry_msgs/Point point   # position in space antenna mast will look at
+---
+# result
+---
+# feedback
```

Comment:
pan and tilt position needed for both result and feedback

Reply:
That is only the case for **PanTitleMoveJoint**, but the unification document has result and feedback empty for this action.

Reply:
Per our discussion, the released OWLAT file is actually wrong, and your action file matches the unification document.  Let's stick with this action file.